src/DataLayer/KeyMotorData/I_KeyMotorData.h

```diff
@@ -19,9 +19,14 @@ class I_KeyMotorData : public QObject
 signals:
     void motorZeroReceived(KeyMotor);
     void motorOneReceived(KeyMotor);
+
+    void motorZeroBusPowerReceived(double);
+    void motorOneBusPowerReceived(double);
+
     void motorSetPowerReceived(double);
     void motorSetCurrentReceived(double);
-    void motorActualSpeedReceived(double);
     void motorBusVoltageReceived(double);
     void motorBusCurrentReceived(double);
+
+    void motorActualSpeedReceived(double);
```

Comment:
Can you re-add this in it's original spot (between `void motorSetCurrentReceived(double);` and `void motorBusVoltageReceived(double);`? That way, it won't show up in the diff.